.github/workflows/devbuild.yml

```diff
@@ -230,6 +243,7 @@ jobs:
 
     env:
       QT_DEBUG_PLUGINS: 1
+      PIP_BREAK_SYSTEM_PACKAGES: 1 # disabling PEP668
```

Comment:
I like the use of an environment variable to control installation behavior and the comment.